gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.jvmargs=-Xmx2048m
+android.useAndroidX=true
+android.enableJetifier=true
+android.enableR8=true
```

Comment:
I'd have a newline here, especially if you will concatenate things to this file.